relay-server/src/actors/outcome.rs

```diff
@@ -296,9 +316,60 @@ impl Message for TrackRawOutcome {
     type Result = Result<(), OutcomeError>;
 }
 
+/// Common implementation for both processing and non-processing
+impl OutcomeProducer {
+    fn send_batch(&mut self, context: &mut Context<Self>) {
+        log::trace!("sending outcome batch");
```

Comment:
_nit_: Move this under the check if `self.unsent_outcomes` is empty and log the batch size, which could be interesting in debugging. Potentially, we'll want to emit a metric for that eventually.

Reply:
ok